### PR TITLE
Add playwright testing library + intial integration tests for the Join Form

### PR DESCRIPTION
**Pull request by WillNilges**
- [x] Set up playwright
- [ ] Add exhaustive Join Form tests
- [x] Configure GitHub action to point at dev instance of MeshDB
- [ ] (optional) Nuke dev instance daily to manage cruft
- [ ] Write docs

### CODE REVIEW

**Comment by WillNilges** [comment]
One day, when I grow up, I want to make these tests run on meshdb repo as well <3

Oh and also k8s-infra because fuck you :)

File: components/JoinForm/JoinForm.tsx [hunk 1]
```diff
@@ -75,7 +76,7 @@ const JoinForm = () => {
       let parsedForm = parseForm(event);
       if (parsedForm === undefined) return;
       // If we were able to glean the form, then save it.
-      recordJoinFormSubmissionToS3(parsedForm);
+      //recordJoinFormSubmissionToS3(parsedForm);
```
Comment (Andrew-Dickinson) [inline components/JoinForm/JoinForm.tsx:79]: Leave a todo here? Why is this commented out

File: components/JoinForm/JoinForm.tsx [hunk 2]
```diff
@@ -121,7 +122,7 @@ const JoinForm = () => {
         <div className={styles.block}>
           <h3>Address Info</h3>
           <input type="text" name="street_address" placeholder="Street Address" required />
-          <input type="text" name="apartment" placeholder="Unit #" required />
+          <input type="text" name="apartment" placeholder="Unit #" />
```
Comment (Andrew-Dickinson) [inline components/JoinForm/JoinForm.tsx:125]: Can we keep this required?

**Comment by Andrew-Dickinson** [comment]
These are a great start on testing, but having PRs on this repo connected to the beta endpoint is not ideal. In theory we'd really like to have unit tests that run on PRs here and just confirm that the generated HTTP request is correct. Then the tests from this PR would be used after PRs are merged, at deploy time to validate that beta is still working in order to gate prod deployments

**Review by Andrew-Dickinson — APPROVED** [review]

**Comment by WillNilges** [comment]
TODO: Check S3 for submission.

**Pull request merged**